edge/pkg/edged/features/features.go

```diff
@@ -0,0 +1,33 @@
+package features
```

Comment:
You can add a bool field(zero value is false) to Edged,  like `DisableCSI`? 
https://github.com/kubeedge/kubeedge/blob/833ba71817960f5c4453b70b29fd87a2c23993f2/staging/src/github.com/kubeedge/api/apis/componentconfig/edgecore/v1alpha2/types.go#L99
```golang
type Edged struct {
    ...
    // DisableCSI disables CSI (Container Storage Interface) support in edgecore.
    // The default value is false, CSI drivers are used for storage operations.
    // Disabling this (set to true) will turn off all CSI-related functionality.
    // This is useful in environments that do not require CSI or for troubleshooting.
    DisableCSI bool `json:"disableCSI"`
}
```